src/6model/reprs/MVMSpeshCandidate.h

```diff
@@ -75,6 +75,9 @@ struct MVMSpeshCandidateBody {
      *  There is a trailing -1 bytecode offset to mark the end of the data.
      */
     MVMint32 *deopt_usage_info;
+
+    MVMint32 *deopt_synths;
```

Comment:
Does this need to be freed in `gc_free`?

Reply:
Or added to `unmanaged_size`?

Reply:
Ah, yes to both. Was so happy to discover the mistake that made it segfault that I forgot that there was still the cleanup to implement.

Reply:
Pushed a new version that addresses your comments. Thanks for the reminder!